From a8f069a98449a81a10d5f6091be2de668f6774a4 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Wed, 1 Jun 2016 19:19:15 +0800
Subject: [PATCH] iommu: dma-iommu: export
 common_iommu_{setup|teardown}_dma_ops

This patch adds EXPORT_SYMBOL_GPL calls to common_iommu_setup_dma_ops
and common_iommu_teardown_dma_ops.

Change-Id: I379d487967587cdb9caffa006a7f3763e6cbfb13
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 drivers/iommu/dma-iommu-ops.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/iommu/dma-iommu-ops.c b/drivers/iommu/dma-iommu-ops.c
index 047c47e3c0ab..862d7823985e 100644
--- a/drivers/iommu/dma-iommu-ops.c
+++ b/drivers/iommu/dma-iommu-ops.c
@@ -456,6 +456,7 @@ bool common_iommu_setup_dma_ops(struct device *dev, u64 dma_base, u64 size,
 
 	return true;
 }
+EXPORT_SYMBOL_GPL(common_iommu_setup_dma_ops);
 
 void common_iommu_teardown_dma_ops(struct device *dev)
 {
@@ -469,3 +470,4 @@ void common_iommu_teardown_dma_ops(struct device *dev)
 
 	arch_set_dma_ops(dev, NULL);
 }
+EXPORT_SYMBOL_GPL(common_iommu_teardown_dma_ops);
-- 
2.35.3

